cli/tools/registry/pm.rs

```diff
@@ -676,12 +676,18 @@ impl AddPackageReq {
         Some((alias, text)) => {
           let (maybe_prefix, entry_text) = parse_prefix(text);
           (
-            maybe_prefix.unwrap_or(Prefix::Jsr),
+            maybe_prefix.ok_or_else(|| {
+              anyhow!("Missing prefix. Possible prefixes are 'jsr:' or 'npm:'")
```

Comment:
I was thinking that if prefix is missing we'd hit both JSR and npm APIs to see if a package with given name exists and suggest concrete registry (or both).